component/common/relabel/relabel.go

```diff
@@ -100,6 +100,13 @@ func (re *Regexp) UnmarshalText(text []byte) error {
 	return nil
 }
 
+// String returns the original string used to compile the regular expression.
+func (re Regexp) String() string {
```

Comment:
This mirrors prometheus' implementation of the same functionality just copied over for our mirrored type